### ISSUE TITLE
App Module: Invalid imports in 1-start branch

### ISSUE THREAD

**Issue by tgavara** [angular-university/angular-universal-course#30]
app.module.ts imports the below even when there is not 'Directives' folder in the '1-start' branch

```
import {AppShellRenderDirective} from "./directives/app-shell-render.directive";
import {AppShellNoRenderDirective} from "./directives/app-shell-norender.directive";
```

**Comment by faridhajnal** [comment]
yes i faced the very same situation. Just go ahead and remove those imports and the directives from the _declarations_ array. If you follow the course, those directives will be created and added to the _app.module.ts_.
Of course this should ideally be fixed by a PR or something so that other students don't experience this issue when freshly cloning the repo
